### DISCUSSION TITLE
Changelog/semantic-releases?

### DISCUSSION THREAD

**Discussion by yuri-scarbaci-lenio** [clauderic/dnd-kit#449] [Ideas]
Any plans on adding changelogs to the library?

I have seen many open source libraries going with https://github.com/semantic-release/semantic-release and we are using it in our internal project with good results

this would help those that are using this lib in production a lot by helping us to track down what the changes are and even more importantly if there is any breaking-change we need to be aware of...

the only trade-off is that the PR/Commits will have to follow a semantic ( something akin to `feat: ...` `fix: ...` , there are more advanced semantics, but this is usually more than enough... )

**Comment by clauderic** [comment]
Each package has its own changelog:

https://github.com/clauderic/dnd-kit/blob/master/packages/core/CHANGELOG.md
https://github.com/clauderic/dnd-kit/blob/master/packages/sortable/CHANGELOG.md
https://github.com/clauderic/dnd-kit/blob/master/packages/accessibility/CHANGELOG.md
https://github.com/clauderic/dnd-kit/blob/master/packages/modifiers/CHANGELOG.md
https://github.com/clauderic/dnd-kit/blob/master/packages/utilities/CHANGELOG.md

The changelogs are auto-generated using [changesets](https://github.com/atlassian/changesets), see https://github.com/clauderic/dnd-kit/pull/342 and https://github.com/clauderic/dnd-kit/tree/master/.changeset

**Reply by yuri-scarbaci-lenio** [reply]
sorry, my bad, I missed that.

Ty for answering and addressing me to those files
